libkbfs/prefetcher.go

```diff
@@ -505,6 +616,8 @@ func (p *blockPrefetcher) run(testSyncCh <-chan struct{}) {
 				pre.req = req
 			}
 
+			p.clearRescheduleState(req.ptr.ID)
```

Comment:
Why do you `delete(p.reschedules, blockID)` in other situations you call this, but not here and below?

Reply:
Because here we are doing the actual prefetch and if it fails, we need to reschedule it with the next backoff duration.  If I deleted the reschedule state here, the backoff timer would start over on every re-fetch.